.github/workflows/stale.yml

```diff
@@ -0,0 +1,76 @@
+---
+name: Close stale issues and PRs
+
+on:
+  schedule:
+    # Every day at noon CEST (10:00 UTC in summer, 11:00 UTC in winter)
+    # Using 10:00 UTC as CEST is UTC+2 (summer time)
+    - cron: '0 10 * * *'
+  workflow_dispatch:
+
+permissions: {}
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    environment:
+      name: main
+    permissions:
+      id-token: write # This is required for getting the required OIDC token from GitHub
+    steps:
+      - uses: DataDog/dd-octo-sts-action@acaa02eee7e3bb0839e4272dacb37b8f3b58ba80 # v1.0.3
+        id: octo-sts
+        with:
+          scope: DataDog/datadog-agent
+          policy: self.stale.manage-stale
+
+      - uses: actions/stale@3a9db7e6a41a89f618792c92c0e97cc736e1b13f # v10.0.0
+        with:
+          repo-token: ${{ steps.octo-sts.outputs.token }}
+
+          # Stale configuration
+          days-before-stale: 15
+          days-before-close: 15
+
+          # Issue configuration
+          stale-issue-message: |
+            This issue has been automatically marked as stale because it has not had recent activity.
+
+
+            It will be closed in 15 days if no further activity occurs. If this issue is still relevant, please add a comment to keep it open.
+
+
+            Thank you for your contributions!
```

Comment:
Suggestion on the copy:

```
This issue has been automatically marked as stale because it has not had any activity in the past 15 days.

It will be closed in 15 days if no further activity occurs. If this issue is still relevant, adding a comment will keep it open. Also, you can always reopen the issue if you missed the window.

Thank you for your contributions!
```